plugin-dev/Scripts/upload-debug-symbols-win.ps1

```diff
@@ -0,0 +1,89 @@
+param([string] $TargetPlatform,[string] $TargetName,[string] $TargetType,[string] $ProjectPath,[string] $PluginPath)
+
+$BinariesPath = "$ProjectPath\Binaries\$TargetPlatform"
+$ConfigPath = "$ProjectPath\Config"
+
+Write-Host "Sentry: Start debug symbols upload"
+
+If ($TargetType -eq "Editor")
+{
+    Write-Host "Sentry: Automatic symbols upload is not required for Editor target. Terminating..."
+    Exit
+}
+
+If ($TargetPlatform -eq "Win64")
+{
+    Set-Variable -Name "CliExec" -Value "$PluginPath\Source\ThirdParty\CLI\sentry-cli-Windows-x86_64.exe"
+}
+Else
+{
+    Write-Host "Sentry: Unexpected platform $TargetPlatform. Terminating..."
+    Exit
+}
+
+function ParseIniFile 
+{  
+    param([parameter(Mandatory = $true)] [string] $filePath)  
+    
+    $anonymous = "NoSection"
+  
+    $ini = @{}  
+    switch -regex -file $filePath  
+    {  
+        "^\[(.+)\]$" # Section  
+        {  
+            $section = $matches[1]  
+            $ini[$section] = @{}  
+            $CommentCount = 0  
+        }  
+
+        "^(;.*)$" # Comment  
+        {  
+            if (!($section))  
+            {  
+                $section = $anonymous  
+                $ini[$section] = @{}  
+            }  
+            $value = $matches[1]  
+            $CommentCount = $CommentCount + 1  
+            $name = "Comment" + $CommentCount  
+            $ini[$section][$name] = $value  
+        }   
+
+        "(.+?)\s*=\s*(.*)" # Key  
+        {  
+            if (!($section))  
+            {  
+                $section = $anonymous  
+                $ini[$section] = @{}  
+            }  
+            $name,$value = $matches[1..2]  
+            $ini[$section][$name] = $value  
+        }  
+    }  
+
+    return $ini  
+}
+
+$ConfigIni = ParseIniFile "$ConfigPath\DefaultEngine.ini"
```

Comment:
Is this file normally under version control? If so, we shouldn't use it for sentry CLI configuration (a.k.a `sentry.properties`) as that's supposed to be excluded from git due to the auth-token.

Reply:
Yes, `DefaultEngine.ini` is an essential part of any UE project that keeps all of its settings (including plugins) and usually it's under source control. Though I don't think it can cause any issues for plugin users since they are likely to take care of securing their project config files. Or did I get it wrong?

Reply:
Even if it's in a private repo, unless it's a single-person project, it's best not to commit auth tokens into source control. Additionally, this disqualifies any open-source games.

Reply:
I'd suggest sticking to a single sentry.propertiies file, regardless of the platform

Reply:
Don't we have the exact same issue with Unity then? AFAIK there auth token is stored in a separate asset (`SentryCliOptions` scriptible object) which apparently could be a part of the game as well

Reply:
It's not part of the game and it's recommended to exclude it from source control as well..

Reply:
> I'd suggest sticking to a single `sentry.propertiies` file, regardless of the platform

We have a template for `sentry.propertiies` file in the repo which is filled with actual user credentials (set in DefaultEngine.ini) when it comes to uploading debug symbols.

Do you mean we should use it both for Windows and other platforms? I recall having some issues with taking this approach on Windows, but I'll double check

Reply:
yes, we use the sentry.properties file on windows without an issue in Unity

Reply:
Practical illustration we shouldn't store the settings in the same settings file - from the in-repo sample PR (#64):

<img width="1391" alt="image" src="https://user-images.githubusercontent.com/6349682/185473721-917178f6-fbf7-4ce7-848b-2e2fabb7089c.png">

Reply:
> How can we move that to a different location so the "shared" file doesn't change?

We can replace user credentials here with a path to `sentry.properties` file which won't be publicly accessible. Later we can add some editor tool that will automate the creation of this file. What do you think?

Reply:
sounds good to me

Reply:
Alright, no more user credentials in `DefaultEngine.ini`. Now in order to obtain auth token and other stuff for Sentry CLI path to `sentry.properties` file has to be provided firs. Apparently not an ideal solution since it may vary on different machines, though at least we've removed the risk of compromising sensitive data